requirements.txt

```diff
@@ -1,8 +1,8 @@
 django>=2.0,<2.2
 swapper>=1.1.0,<1.2.0
 openwisp-utils>=0.2,<0.3
-djangorestframework>=3.8.2,<3.9
+djangorestframework>=3.8.2,<3.9.0
```

Comment:
the latest version of DRF is 3.9.0, hence this will not catch it. increase it to one minor version, `3.10.0`

Reply:
Please check it now :) I've made the changes. Please let me know if I need to change anything else as I'm unable to submit other tasks that I've already completed. 
Thanks Sir